mempool/lanes_info.go

```diff
@@ -1,64 +1,62 @@
 package mempool
 
 import (
-	"slices"
-
 	"github.com/cometbft/cometbft/types"
 )
 
-type LanesInfo struct {
-	lanes       []types.Lane
-	defaultLane types.Lane
+type LaneData struct {
```

Comment:
With this renaming there are variable names, file names and comments that need to be updated because they are inconsistent. 

I think it makes sense that on the application and the protos we have lane "data" that is converted to lane "info" when imported into Comet, but the names (and all the rest) have to be consistent.